composer.json

```diff
@@ -44,7 +44,9 @@
         "symfony/yaml": "~2.3|~3.0",
         "symfony/security-acl": "~2.4|~3.0",
         "sonata-project/intl-bundle": "dev-master",
-        "symfony/phpunit-bridge": "~2.7|~3.0"
+        "symfony/phpunit-bridge": "~2.7|~3.0",
```

Comment:
That was because of a merge after rebase, should I delete it? With my commit, it would be already resolved...